client/src/components/pages/application/Tasks/index.css

```
@@ -1,53 +1,110 @@
-.tasks-container {
-  margin-top: 15vh;
-  margin-left: 25vw;
-  margin-bottom: 10vh;
-  width: 60vw;
+.Task_page {
+  background-color: rgb(245, 245, 245);
+  width: 100%;
+  min-height: 99vh;
   display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-}
-.task_container_field {
-  display: flex;
-  flex-direction: row;
+  justify-content: center;
   align-items: center;
+  padding-top: 3%;
 }
-.task_container_field input {
-  height: 0.3rem;
-  background-color: rgb(255, 255, 255);
-  width: 30rem;
-}
-.tasks-title {
-  color: rgb(255, 123, 0);
-}
-
-.tasks-background {
-  position: fixed;
-  z-index: -1;
-  top: 20%;
-  height: 70vh;
-  right: 2%;
-}
-
 .tasks_buttons {
-  width: 25rem;
-  margin-top: 3rem;
-  margin-left: 15vw;
   display: flex;
   justify-content: space-around;
+  margin: 2rem;
+}
+.Task_container_page {
+  background-color: rgba(245, 245, 245, 0.411);
+  display: flex;
+  flex-direction: column;
+  padding-right: 4%;
+  width: 40rem;
+  z-index: 1;
 }
-
 .tasks-paragraph {
   margin-left: 2.5rem;
   margin-bottom: 2rem;
-  background-color: white;
+  background-color: rgba(245, 245, 245, 0.411);;
```

delete double semicolons 
```suggestion
  background-color: rgba(245, 245, 245, 0.411);
```